data/ui/oops-window.ui

```diff
@@ -205,455 +188,251 @@
               </packing>
             </child>
             <child>
-              <object class="GtkBox" id="box_panel_right">
+              <object class="GtkStack" id="nb_problem_layout">
                 <property name="visible">True</property>
-                <property name="can_focus">False</property>
-                <property name="orientation">vertical</property>
+                <property name="expand">True</property>
+                <property name="margin">18</property>
                 <child>
-                  <object class="GtkNotebook" id="nb_problem_layout">
+                  <object class="GtkBox" id="gd_problem_info">
                     <property name="visible">True</property>
-                    <property name="can_focus">False</property>
-                    <property name="hexpand">True</property>
-                    <property name="vexpand">True</property>
-                    <property name="show_tabs">False</property>
-                    <property name="show_border">False</property>
+                    <property name="orientation">vertical</property>
+                    <property name="spacing">18</property>
                     <child>
-                      <object class="GtkBox" id="box3">
+                      <object class="GtkBox">
                         <property name="visible">True</property>
-                        <property name="can_focus">False</property>
-                        <property name="orientation">vertical</property>
+                        <property name="spacing">12</property>
+                        <child>
+                          <object class="GtkImage" id="img_app_icon">
+                            <property name="visible">True</property>
+                            <property name="icon_name">image-missing-symbolic</property>
+                          </object>
+                        </child>
                         <child>
-                          <object class="GtkScrolledWindow" id="scrolledwindow2">
+                          <object class="GtkBox">
                             <property name="visible">True</property>
-                            <property name="can_focus">True</property>
-                            <property name="hscrollbar_policy">never</property>
-                            <property name="vscrollbar_policy">automatic</property>
+                            <property name="hexpand">True</property>
+                            <property name="orientation">vertical</property>
+                            <property name="margin-top">12</property>
+                            <child>
+                              <object class="GtkLabel" id="lbl_reason">
+                                <property name="visible">True</property>
+                                <property name="xalign">0</property>
+                                <property name="label">Application killed by signal</property>
```

Comment:
Do we need any initial value of the label here? I know this is not anything you are adding now, just reformatting the content which existed before. I'm just thinking that maybe it's time to remove.

Reply:
I’ll take a stab at that, sure.